### PR TITLE
Add 'run_test' configs to skip ffmpeg input test on startup

### PR DESCRIPTION
**Pull request by jjlawren**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17769

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/binary_sensor.ffmpeg_motion.markdown [hunk 1]
```diff
@@ -72,6 +72,11 @@ extra_arguments:
   description: Extra options to pass to `ffmpeg`, e.g., video denoise filtering.
   required: false
   type: string
+run_test:
+  description: Check if `input` is usable by ffmpeg.
+  required: false
+  default: True
```
Comment (frenck) [inline source/_components/binary_sensor.ffmpeg_motion.markdown]:
```suggestion
  default: true
```

File: source/_components/binary_sensor.ffmpeg_noise.markdown [hunk 2]
```diff
@@ -66,6 +66,11 @@ output:
   description: Allows you to send the audio output of this sensor to an Icecast server or other FFmpeg-supported output, e.g., to stream with Sonos after a state is triggered.
   required: false
   type: string
+run_test:
+  description: Check if `input` is usable by ffmpeg.
+  required: false
+  default: True
```
Comment (frenck) [inline source/_components/binary_sensor.ffmpeg_noise.markdown]:
```suggestion
  default: true
```

File: source/_components/camera.ffmpeg.markdown [hunk 3]
```diff
@@ -39,6 +39,11 @@ extra_arguments:
   description: Extra options to pass to `ffmpeg`, e.g., image quality or video filter options.
   required: false
   type: string
+run_test:
+  description: Check if `input` is usable by ffmpeg.
+  required: false
+  default: True
```
Comment (frenck) [inline source/_components/camera.ffmpeg.markdown]:
```suggestion
  default: true
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Really small change needed there @jjlawren!
Could you take a look? 👍

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by jjlawren** [comment]
Changed some functionality in the parent PR to address design concerns, updated docs to match.

**Comment by jjlawren** [comment]
Closing in favor of #7342.

**Pull request closed without merging**